packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.tsx

```diff
@@ -179,8 +179,9 @@ export class SwatchColorPicker extends BaseComponent<ISwatchColorPickerProps, IS
   @autobind
   private _onRenderColorOption(colorOption: IColorCellProps): JSX.Element {
     // Build an SVG for the cell with the given shape and color properties
+    let color = colorOption.color as string;
```

Comment:
What is with this typecast?

Reply:
Color could be of type undefined so we have to do a typecast for it so that it's guaranteed to be a string.

This was originally used in our code, but, because we needed to use it in 2 different places, so I decided to store it in a variable.

Reply:
A typecast doesn't guarantee it to be a string, it could actually be undefined. You should instead put the code that expects it to be a string behind an if check for undefined

Reply:
The typecast is still there, see the conversation on #3432, where we are looking at making color a required prop.